tests/test_indexer.py

```diff
@@ -78,7 +82,6 @@ def tearDownClass(cls):
             cls.mock_s3.stop()
 
     def setUp(self):
-        Config.set_config(BucketStage.TEST)
```

Comment:
This is already being called in setUpClass, which is sufficient, and shouldn't be done again in setUp.